### PR TITLE
Modified Java Blob Store to make use of Storage class

### PR DESCRIPTION
**Pull request by kirkeaton**
Resolves https://github.com/nanoPayinc/NANOPAY/issues/1992

1. Modified Java Blob Store to make use of Storage class
2. Removed root property from Java BlobStore

### CODE REVIEW

File: src/foam/blob/BlobJava.js [hunk 1]
```diff
@@ -190,21 +190,21 @@ foam.CLASS({
...
-    'java.io.FileOutputStream'
+    'java.io.FileOutputStream',
+    'foam.nanos.fs.Storage'
   ],
 
   constants: [
     {
-      name: 'BUFFER_SIZE',
       value: 4096,
+      name: 'BUFFER_SIZE',
```
Comment (kgrgreer) [inline src/foam/blob/BlobJava.js]: Why switch the order? I think it's more conventional to have the name before the value.

**Review by jlhughes — APPROVED** [review]

File: src/foam/blob/Blob.js [hunk 2]
```diff
@@ -477,7 +477,8 @@ foam.CLASS({
   properties: [
     {
       class: 'String',
-      name: 'root'
+      name: 'root',
```
Comment (kgrgreer) [inline src/foam/blob/Blob.js:480]: Is this property still used? Can you document what it's for?

File: src/foam/blob/BlobJava.js (hunk 1 above)
Comment (kirkeaton) [reply src/foam/blob/BlobJava.js]: i don't recall changing this

File: src/foam/blob/Blob.js (hunk 2 above)
Comment (kirkeaton) [reply src/foam/blob/Blob.js:480]: Yes it is, yes I can

**Review by jlhughes — APPROVED** [review]

**Pull request merged**